ReactiveXComponentTest/ReactiveXComponentTest.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="NUnit" Version="3.11.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.12.0" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.5.0" />
+    <PackageReference Include="System.Reactive.Linq" Version="3.1.1" />

```

Comment:
Why do you add this reference ?

Reply:
I need this when i add <privateAssets>all</PrivateAssets> in ReactiveXComponent.csproj

Reply:
PrivateAsset = all